handlers/main.yml

```diff
@@ -11,7 +11,7 @@
 
 - name: Restart Logviewer
   service: name=storm-logviewer state=restarted
-  when: storm_supervisor_enabled
+  when: storm_logviewer_enabled
   when: run_mode == 'Deploy'
```

Comment:
This looks good, still need to test but would you mind combining the when conditionals for all these tasks?